public/reorderSiteAwards.php

```diff
@@ -10,46 +10,171 @@
 RenderContentStart("Reorder Site Awards");
 ?>
 <script>
-function updateAwardDisplayOrder(awardType, awardData, awardDataExtra, objID) {
-    var inputText = $('#' + objID).val();
-    var inputNum = Math.max(-1, Math.min(Number(inputText), 10000));
+let currentGrabbedRow = null;
+
+function updateAllAwardsDisplayOrder() {
     showStatusMessage('Updating...');
-    $.post('/request/user/update-site-award.php', {
-        type: awardType,
-        data: awardData,
-        extra: awardDataExtra,
-        number: inputNum,
+
+    const awards = [];
+
+    $('.displayorderedit').each(function (index, element) {
+        const row = $(element).closest('tr');
+        const awardType = row.find("input[type='hidden'][name='type']").val();
+        const awardData = row.find("input[type='hidden'][name='data']").val();
+        const awardDataExtra = row.find("input[type='hidden'][name='extra']").val();
+        const displayOrder = parseInt($(element).val(), 10);
+
+        awards.push({
+            type: awardType,
+            data: awardData,
+            extra: awardDataExtra,
+            number: displayOrder
+        });
+    });
+
+    $.post('/request/user/update-site-awards.php', { awards })
+        .done(function (response) {
+            showStatusMessage('Awards updated successfully');
+            $('#rightcontainer').html(response.updatedAwardsHTML);
+        })
+        .fail(function () {
+            showStatusMessage('Error updating awards');
+        });
+}
+
+function handleRowDragStart(event) {
+    currentGrabbedRow = event.target;
+    event.target.style.opacity = '0.3';
```

Comment:
I'm not sure if there's an easy solution, but trying to highlight an existing order value by click/dragging in the editable field starts a drag action without highlighting the text.

Reply:
I experienced this as well and tried multiple approaches to mitigate it, but was unable to land on a solution.

Reply:
Based on the other feedback here I am going to experiment with getting rid of the Display Order field altogether. We may be able to use something like a checkbox for if the user wants to hide an award.

For group ordering, we can have something on the page for the user to manually determine which order they'd like their groups in. This doesn't seem too bad given there's only three of them.

Reply:
If we are getting rid of the display order field would it be possible to change the way the awards are displayed in the re-ordering section to be more like what they show in the right column so that users can see exactly what it will look like as they re-order. This also seems like it would be easier to update as a user vs a long column list of single achievements per line, users with hundreds of awards would take a while to scroll up but with 5 per row would be 5x less scrolling and 5x more visibility. Not sure how hiding awards would work in this case though, possibly clicking on the award toggles it's visibility or moving it to a separate hidden section above the current group?

Reply:
I agree that's where we ultimately want to be. I've considered what it would take to get us there on a first pass and, to your point, there's enough nuance with hiding awards that this becomes a much more complex change. 

I think what you're describing is ultimately where we want to land: a WYSIWYG for award reordering. But there is enough to hash out with award hiding that I'd rather make that adjustment in a subsequent PR. I think we're more likely to be successful if we get drag and drop working first, then perfect it.

Reply:
This should now be resolved given that the input fields have been removed altogether. Initiating a drag from the dragging on the buttons is still possible, but I don't think it will cause confusion.